WalletWasabi/WabiSabi/Crypto/WabiSabiClient.cs

```diff
@@ -107,6 +107,7 @@ public class WabiSabiClient
 			{
 				credentialAmountsToRequest.Add(0);
 			}
+			credentialAmountsToRequest.Shuffle();
```

Comment:
Why?

Reply:
I am not 100% sure this is strictly necessary but I am sure it is not a mistake. The thing is that if well the coordinator cannot see the value that is requested, the fact that the first credential request is almost always the one that contains the non-zero value requested and that in some cases that amount is exactly the same as `delta` could provide clues to a malicious coordinator.

Reply:
This shouldn't be a concern unless something has gone horribly wrong with the unlinkability of credentials, but it doesn't hurt either.

Reply:
What do you say, guys? Should I revert it?

Reply:
I can't make a good argument one way or another, but when in doubt/feeling agnostic I prefer deleting code over not deleting it by default.

Reply:
Removed.